src/Sdk/PasswordlessOptions.cs

```diff
@@ -1,10 +1,25 @@
 namespace Passwordless.Net;
 
+/// <summary>
+/// Represents all the options you can use to configure a backend Passwordless system.
+/// </summary>
 public class PasswordlessOptions
 {
+    /// <summary>
+    /// Passwordless Cloud Url
+    /// </summary>
     public const string CloudApiUrl = "https://v4.passwordless.dev";
 
-    public string ApiKey { get; set; } = default!;

```

Comment:
* Took ApiKey out of PasswordlessOptions since those options will be purely back-end options for Passwordless.Net

Not sure I agree on this change. For Razor Pages/SDK apps, they probably want to stick the value in the options (where they configure "passwordless") and use it in the their razor pages.  Co-locating configuration feels like good DX.

Reply:
Optional: Only used for frontend operations.